recipes/scispacy/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "scispacy" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scispacy-{{ version }}.tar.gz
+  sha256: a89e2de55714d4636e22429508c1140c3877c8e614839c5073506ed78c468551
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - conllu
+    - joblib
+    - nmslib >=1.7.3.6
+    - numpy
+    - pysbd
+    - python >=3.6
+    - requests >=2.0.0,<3.0.0
+    - scikit-learn >=0.20.3
+    - spacy >=3.0.0,<3.1.0
+
+test:
+  imports:
+    - scispacy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://allenai.github.io/SciSpaCy/
+  summary: A full SpaCy pipeline and models for scientific/biomedical documents.
+  license: Apache-2.0
```

One file is not licensed under the repo license: https://github.com/allenai/scispacy/blob/main/scispacy/hearst_patterns.py

This either needs to be manually included or the package needs to be updated and the file excluded. 

You can include it here by adding the following to your meta.yaml:
```suggestion
  license: Apache-2.0 AND BSD-3-Clause
```
Additionally, the raw license file should also be added to this package.